internal/ethapi/api.go

```diff
@@ -2729,6 +2729,12 @@ func (api *DebugAPI) GetTraceStack() string {
 	}
 }
 
+// PeerStats returns the current head height and td of all the connected peers
+// along with few additional identifiers.
+func (api *DebugAPI) PeerStats() interface{} {
```

Comment:
Falling back to returning `interface{}` because I Can't use `eth` package due to cyclic dependency. Happy to move the struct defined elsewhere if needed